vizro-core/tests/unit/vizro/models/_components/test_graph.py

```diff
@@ -110,6 +193,21 @@ class MyGraph(vm.Graph):
         assert my_graph.actions == []
 
 
+class TestGraphInstantiationJson:
+    @pytest.mark.parametrize(
+        "target",
+        [
+            "scatter",
+            "tests.unit.vizro.models._components.test_graph.custom_scatter",
```

Comment:
```suggestion
            f"{__name__}.test_graph.custom_scatter",
```
Feels nicer than harcdoding the file path. Then we can also have a case for something that we know will work and will test the generic import path case rather than the `from vizro.plotly.express` one:
```suggestion
            f"{__name__}.test_graph.custom_scatter",
            "vizro.plotly.express.scatter",
```

Reply:
It works, see new test!